packages/features/onboarding/lib/onboarding-path.service.ts

```diff
@@ -0,0 +1,24 @@
+import { FeaturesRepository } from "@calcom/features/flags/features.repository";
+import type { PrismaClient } from "@calcom/prisma";
+
+export class OnboardingPathService {
+  static async getGettingStartedPath(prisma: PrismaClient): Promise<string> {
+    const featuresRepository = new FeaturesRepository(prisma);
+    const onboardingV3Enabled = await featuresRepository.checkIfFeatureIsEnabledGlobally("onboarding-v3");
+    return onboardingV3Enabled ? "/onboarding/getting-started" : "/getting-started";
+  }
```

Comment:
Resuable util for creating the correct gettingStarted path based on feature flag.